pandapower/shortcircuit/currents.py

```diff
@@ -147,7 +147,7 @@ def _current_source_current(net, ppci, bus_idx):
     # _is_elements_final exists for some reason, and weirdly it can be different than _is_elements. 
     # it is not documented anywhere why it exists and I don't have any time to find out, but this here fixes the problem.
 
-    if np.alltrue(net.sgen.current_source.values):
+    if all(net.sgen.current_source.values):
```

Comment:
If I see correctly, all is not imported from numpy here, I would use numpy.all

Reply:
Thank you, i changed it.